tck/features/clauses/return/Return2.feature

```diff
@@ -0,0 +1,301 @@
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Return2 - Return single expression (correctly projecting an expression)
+
+  Scenario: Returning a node property value
+    Given an empty graph
+    And having executed:
+      """
+      CREATE ({num: 1})
+      """
+    When executing query:
+      """
+      MATCH (a)
+      RETURN a.num
+      """
+    Then the result should be, in any order:
+      | a.num |
+      | 1     |
+    And no side effects
+
+  Scenario: Missing node property should become null
+    Given an empty graph
+    And having executed:
+      """
+      CREATE ({num: 1})
+      """
+    When executing query:
+      """
+      MATCH (a)
+      RETURN a.name
+      """
+    Then the result should be, in any order:
+      | a.name |
+      | null   |
+    And no side effects
+
+  Scenario: Returning a relationship property value
+    Given an empty graph
+    And having executed:
+      """
+      CREATE ()-[:T {num: 1}]->()
+      """
+    When executing query:
+      """
+      MATCH ()-[r]->()
+      RETURN r.num
+      """
+    Then the result should be, in any order:
+      | r.num |
+      | 1     |
+    And no side effects
+
+  Scenario: Missing relationship property should become null
+    Given an empty graph
+    And having executed:
+      """
+      CREATE ()-[:T {name: 1}]->()
+      """
+    When executing query:
+      """
+      MATCH ()-[r]->()
+      RETURN r.name2
+      """
+    Then the result should be, in any order:
+      | r.name2 |
+      | null    |
+    And no side effects
+
+  Scenario: Adding a property and a literal in projection
+    Given an empty graph
+    And having executed:
+      """
+      CREATE ({num: 1})
+      """
+    When executing query:
+      """
+      MATCH (a)
+      RETURN a.num + 1 AS foo
+      """
+    Then the result should be, in any order:
+      | foo |
+      | 2   |
+    And no side effects
+
+  Scenario: Returning a list property
+    Given an empty graph
+    And having executed:
+      """
+      CREATE ({numbers: [1, 2, 3]})
+      """
+    When executing query:
+      """
+      MATCH (n)
+      RETURN n
+      """
+    Then the result should be, in any order:
+      | n                      |
+      | ({numbers: [1, 2, 3]})
```

Comment:
```suggestion
      | ({numbers: [1, 2, 3]}) |
    And no side effects
```

Should fix cucumber parsing problem and InvalidFeatureFormatException during build.